### ISSUE TITLE
UI crashes after first file

### ISSUE THREAD

**Issue by vampjoseph** [GameMasterJack/Crunchyroll-Anime-Downloader-UI#2]
This is regarding the latest files on the main page. The UI crashes when I try to batch download a show from CR. It doesn't matter the show, it downloads the first file specified and then after conversion the window closes even though there are more to be downloaded. I would include an error code but CMD prompt closes before I can see what it is and I can't find an error log.

**Comment by GameMasterJack** [comment]
Start command prompt. type "cd [UI_Folder_Path]" Run the UI program. Try to repeat error message. It should go back to the cmd screen and copy it from there.

**Comment by vampjoseph** [comment]
I got this issue figured out. The settings file deleted the language setting from itself for some reason and I was forgetting to put a "p" at the end of the video quality in cmd.

**Comment by vampjoseph** [comment]
though I'm having a slight problem (not really with the ui but a problem nonetheless and with darkztar gone without a trace I don't really know what to do). Every time I download a file, regardless of the show and regardless of whether I use the ui or copy/paste the link. the file that is downloading is not labelled as the episode. instead it is labelled Login (same thing with the .ass file). It used to work with minor hiccups but now it's wierd. Also the episodes I'm downloading are not premium episodes (though even if they were, I'd just get a snippet that would still be labelled correctly)

**Comment by GameMasterJack** [comment]
I believe that all you have to do is type "Login". Enter your username and
password (It may ask for it two times). It should log you in.
Anyways, I apologize it's not a better program but I made it mostly for
myself :P

Also that's really weird that its being named that because there isnt
anything in the source code that would cause that. Have you edited the
files in anyway?

**Comment by Jakh33** [comment]
Been having these issue myself for the last few days, suddenly all files I download are called "login". Batch downloads have the problem that the next episode is also called login and therefore overwrites the last one.

**Comment by vampjoseph** [comment]
the only file that I've edited is the settings file. I make sure that the quality is what I want (720p) and the language (english). That usually solves the problem with batch downloading. Also I've tried making a new cookies file with no success (meaning that I have logged in with both the ui and the make_cookies.bat and _make_cookies.bat file). I'm not good at programming but maybe cr has changed the backend a bit? just a thought.

EDIT: Just reinstalled python thinking that was the problem. No dice.
